src/main/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePass.scala

```diff
@@ -64,8 +67,12 @@ class PropertiesFilePass(cpg: Cpg, projectRoot: String) extends ForkJoinParallel
     } match {
       case Success(keyValuePairs) =>
         val propertyNodes = keyValuePairs.map(addPropertyNode(_, builder))
-        propertyNodes.foreach(connectGetPropertyLiterals(_, builder))
-        connectAnnotatedParameters(propertyNodes, builder)
+
+        propertyNodes.foreach(propertyNode => {
+          connectGetPropertyLiterals(_, builder)
```

Comment:
can you pass `propertyNode` instead of `_` to make the code more readable and consistent with other method calls?